src/org/zaproxy/zap/extension/fuzz/payloads/generator/RegexPayloadGenerator.java

```diff
@@ -181,15 +181,41 @@ private static boolean isInfiniteImpl(Generex generator, int limit) {
      * @return the number of payloads that would be produced by the given regular expression
      * @throws IllegalArgumentException if the given {@code regex} is {@code null} or not valid
      * @see #DEFAULT_LIMIT_CALCULATION_PAYLOADS
+     * @see #calculateNumberOfPayloads(String, int, boolean)
      * @see #isInfinite(String, int)
      * @see #isValid(String)
      */
     public static int calculateNumberOfPayloads(String regex, int limit) {
+        return calculateNumberOfPayloads(regex, limit, false);
+    }
+
+    /**
+     * Calculates the number of payloads that the given regular expression would produce, limiting up to the given {@code limit}
+     * (if positive) and whether its random.
```

Comment:
`it's`

This is a weird English thing where "it is" would be contracted as "it's". But, any other time an apostrophe s is possessive (like Fred's book). One step further "its" is the possessive for it (if it owns something)..... straight forward eh?

Reply:
Right, that was a typo.